homeassistant/components/pushetta/notify.py

```diff
@@ -1,17 +1,17 @@
 """Pushetta platform for notify component."""
 import logging
 
+from pushetta import Pushetta, exceptions as PushettaExceptions
```

Comment:
Module names are normally lowercase snakecase.

https://www.python.org/dev/peps/pep-0008/#package-and-module-names

Reply:
Good idea to stick with the python conventions! Fixed in https://github.com/home-assistant/home-assistant/pull/29332/commits/eb22aad8adf474272bb237b9df8a2b1e52615c4d